src/servicebus/azext_servicebus/_utils.py

```diff
@@ -0,0 +1,35 @@
+from azure.mgmt.servicebus.models.service_bus_management_client_enums import SkuName, SkuTier, AccessRights
+
+
+def skunameconverter(skuname):
+
+    if skuname == 'Basic':
+        return SkuName.basic
+    if skuname == 'Standard':
+        return SkuName.standard
+    if skuname == 'Premium':
+        return SkuName.premium
+
+
+def skutireconverter(skutire):
+
+    if skutire == 'Basic':
+        return SkuTier.basic
+    if skutire == 'Standard':
+        return SkuTier.standard
+    if skutire == 'Premium':
+        return SkuTier.premium
+
+
+def accessrights_converter(accessrights):
+    if len(accessrights) > 0:
+        accessrights_new = []
+        for index in accessrights:
+            if index == 'Send':
+                accessrights_new.append(AccessRights.send)
+            if accessrights[index] == 'Manage':
+                accessrights_new.append(AccessRights.manage)
+            if accessrights[index] == 'Listen':
+                accessrights_new.append(AccessRights.listen)
```

Comment:
Sometimes you do `index` and elsewhere `accessrights[index]`.
If it's an index, only one of them will work. If it's a string, only one of them will work.
Don't see how both of them can work.